spring-xd-spark-streaming-tests/src/test/java/org/springframework/xd/spark/streaming/KafkaTransportSparkStreamingTests.java

```diff
@@ -31,16 +27,11 @@
 public class KafkaTransportSparkStreamingTests extends AbstractSparkStreamingTests {
 
 	@ClassRule
-	public static KafkaTestSupport kafkaTestSupport;
+	public static KafkaTestSupport kafkaTestSupport = new KafkaTestSupport();
 
 	static {
-		kafkaTestSupport = new KafkaTestSupport();
-		Properties brokerConfigProperties = TestUtils.createBrokerConfig(0, TestUtils.choosePort());
-		kafkaTestSupport.setBrokerConfig(brokerConfigProperties);
-		System.setProperty(KafkaConnectionPropertyNames.KAFKA_BROKERS,
-				brokerConfigProperties.getProperty("host.name") + ":" + brokerConfigProperties.getProperty("port"));
-		System.setProperty(KafkaConnectionPropertyNames.KAFKA_ZK_ADDRESS,
-				brokerConfigProperties.getProperty("zookeeper.connect"));
+		System.setProperty(KafkaConnectionPropertyNames.KAFKA_BROKERS, kafkaTestSupport.getBrokerAddress());
+		System.setProperty(KafkaConnectionPropertyNames.KAFKA_ZK_ADDRESS, kafkaTestSupport.getZkConnectString());
 	}
```

Comment:
Why is this stuff done in a static block instead of using @BeforeClass.  That way you could clean it up via @AfterClass?